plugins/com.google.cloud.tools.eclipse.appengine.validation/src/com/google/cloud/tools/eclipse/appengine/validation/AppEngineWebXmlValidator.java

```diff
@@ -42,9 +42,7 @@ protected void validate(IResource resource, byte[] bytes)
       Map<BannedElement, Integer> bannedElementOffsetMap =
```

Comment:
separate issue, but I just realized that since we're using BannedElement as a key in a map, we should give it equals and hashCode methods: https://github.com/GoogleCloudPlatform/google-cloud-eclipse/issues/1507